### PR TITLE
Mobile controls

### LINKED ISSUE #15: Add control buttons for mobile devices
```
     [-------------]
     [  Stage  ]
     [-------------]
  _
_|  |_         (A)
_   _|
 | |
```

### PR DESCRIPTION
**Pull request by towerofnix**
Demo screenshot

It could use some work - if you play around with this for a while on Safari (iOS), you'll definitely realize that gestures and whatnot cause the page to zoom a bit while using controls.. it's annoying, but this is a start.

Fixes #15.

### CODE REVIEW

**Comment by towerofnix** [comment]
To collaborators - do not merge yet!

**Comment by nanalan** [comment]
+1

**Comment by nanalan** [comment]
Pretty sure you've forgotten to push `controls/controller.js`.

**Comment by dekrain** [comment]
No, it is in `src/contols/contolles.js`.

**Comment by dekrain** [comment]
Yes, It's because gulp changes `~~~~\n~~~~` to `~~~~\r\n~~~~`

File: index.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html lang='en'>
 
 <meta charset='utf-8'>
+<meta name='viewport' content='initial-scale=1, maximum-scale=1, width=device-width, user-scalable=no'
```
Comment (towerofnix) [inline index.html:5]: Have you used iOS 10 though? IIRC this does like nothing :(

Comment (bates64) [reply index.html:5]: Yep, I hate iOS 10 :stuck_out_tongue_winking_eye:

Comment (nanalan) [inline index.html]: Yep, I hate iOS 10 :stuck_out_tongue_winking_eye:

**Comment by towerofnix** [comment]
Closing because this is very very old and I don't plan more work on slw (such as figuring out how to merge this).

**Pull request closed without merging**